src/config.ts

```diff
@@ -0,0 +1,14 @@
+import bytes from 'bytes';
+
+export type Config = typeof config;
+
+export const config = {
+  // max size of data that can be provided with a RecordsWrite
+  maxRecordDataSize      : bytes(process.env.MAX_RECORD_DATA_SIZE || '1gb'),
+  // port that server listens on
+  port                   : parseInt(process.env.DS_PORT || '3000'),
+  // whether to enable 'ws:'
+  webSocketServerEnabled : { 'on': true, 'off': false}[process.env.DS_WEBSOCKET_SERVER] || true
```

Comment:
this statement will always evaluate to `true`:
`{ 'on': true, 'off': false}[process.env.DS_WEBSOCKET_SERVER] || true`

If `DS_WEBSOCKET_SERVER` is intended to be a required env-var, the line could be changed it to:
`  webSocketServerEnabled : { 'on': true, 'off': false}[process.env.DS_WEBSOCKET_SERVER] || true`

If  `DS_WEBSOCKET_SERVER` is intended to be an optional env-var, then a one-liner option is:
`  webSocketServerEnabled : process.env.DS_WEBSOCKET_SERVER !== 'off'`

Reply:
good catch kind sir. thx